src/Pickles/Pickles.DocumentationBuilders.Json.UnitTests/AutomationLayer/StepDefinitions.cs

```diff
@@ -66,6 +66,10 @@ public void ThenTheResultShouldBe(string expectedResult)
             var actualResult = this.FileSystem.File.ReadAllText(@"c:\output\pickledFeatures.json");
             actualResult = actualResult.Replace("{", "{{").Replace("}", "}}");
             expectedResult = expectedResult.Replace("{", "{{").Replace("}", "}}");
+
+            actualResult = Regex.Replace(actualResult, @"\s+", string.Empty);
```

Comment:
My regular expressions are rusty - what does this do?

Reply:
@dirkrombauts: i realized that the unit test failed not because json content but different white spaces in the json (probably white spaces coding between operating systems/ide's) - this regular expression remove all white spaces from the actual and expected json results - thanks that out check will validate actual content (and does not care about tabs and spaces coding anymore).